webpack/ForemanTasks/Components/TaskDetails/Components/Task.js

```diff
@@ -47,96 +52,109 @@ const Task = props => {
-                href={`/foreman_tasks/tasks/${id}/sub_tasks`}
+                href={`/foreman_tasks/dynflow/${externalId}`}
+                disabled={!dynflowEnableConsole}
+                rel="noopener noreferrer"
+                target="_blank"
               >
-                {__('Sub tasks')}
+                {__('Dynflow console')}
               </Button>
-            )}
-            <Button
-              className="unlock-button"
-              bsSize="small"
-              disabled={state !== 'paused'}
-              onClick={unlockModalActions.setModalOpen}
-            >
-              {__('Unlock')}
-            </Button>
-            <Button
-              className="force-unlock-button"
-              bsSize="small"
-              disabled={state === 'stopped'}
-              onClick={forceUnlockModalActions.setModalOpen}
-            >
-              {__('Force Unlock')}
-            </Button>
-          </Col>
+              <Button
+                className="resume-button"
+                bsSize="small"
+                disabled={!resumable}
+                onClick={() => {
+                  if (!taskReload) {
+                    taskProgressToggle();
+                  }
+                  resumeTaskRequest(id, action);
+                }}
+              >
+                {__('Resume')}
+              </Button>
+              <Button
+                className="cancel-button"
+                bsSize="small"
+                disabled={!cancellable}
+                onClick={() => {
+                  if (!taskReload) {
+                    taskProgressToggle();
+                  }
+                  cancelTaskRequest(id, action);
+                }}
+              >
+                {__('Cancel')}
+              </Button>
+              {parentTask && (
+                <Button
+                  className="parent-button"
+                  bsSize="small"
+                  href={`/foreman_tasks/tasks/${parentTask}`}
+                >
+                  {__('Parent task')}
+                </Button>
+              )}
+              {hasSubTasks && (
+                <Button
+                  className="subtask-button"
+                  bsSize="small"
+                  href={`/foreman_tasks/tasks/${id}/sub_tasks`}
+                >
+                  {__('Sub tasks')}
+                </Button>
+              )}
+              <Button
+                className="unlock-button"
+                bsSize="small"
+                disabled={state !== 'paused'}
+                onClick={unlockModalActions.setModalOpen}
+              >
+                {__('Unlock')}
+              </Button>
+              <Button
+                className="force-unlock-button"
+                bsSize="small"
+                disabled={state === 'stopped'}
+                onClick={forceUnlockModalActions.setModalOpen}
+              >
+                {__('Force Unlock')}
+              </Button>
+            </Col>
+          )}
```

Comment:
since this section grew really big, can you move all buttons into another component?

Reply:
Yeah good idea